sdk/cosmosdb/ci.yml

```diff
@@ -0,0 +1,32 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
+
+trigger:
+  branches:
+    include:
+    - master
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/cosmosdb/
+
+pr:
+  branches:
+    include:
+    - master
+    - feature/*
+    - hotfix/*
+    - release/*
+  paths:
+    include:
+    - sdk/cosmosdb/
+
+stages:
```

Comment:
Looks like the stages tag does not work. Use 
`
extends:
  template:
`
Please see existing ci.yml for reference.